org.jvoicexml.config/src/main/java/org/jvoicexml/config/ConfigurationFolderMonitor.java

```diff
@@ -153,6 +153,7 @@ public void run() {
                         LOGGER.warn(e.getMessage(), e);
                     }
                 }
+                key.reset(); // Reset to await further events on the file
```

Comment:
Resolves a defect uncovered by the TestConfigurationFolderMonitor. Without this, only the first change to a file is registered. This resets the watch on the file so that subsequent events are captured.